src/main/java/be/seeseemelk/mockbukkit/WorldBorderMock.java

```diff
@@ -0,0 +1,166 @@
+package be.seeseemelk.mockbukkit;
+
+import org.apache.commons.lang.Validate;
+import org.bukkit.Location;
+import org.bukkit.Server;
+import org.bukkit.World;
+import org.bukkit.WorldBorder;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.util.BoundingBox;
+import org.jetbrains.annotations.NotNull;
+
+public class WorldBorderMock implements WorldBorder
+{
+
+	private final World world;
+	private final Server server;
+	private double size = 6.0E7;
+	private double damageAmount = 0.2;
+	private double damageBuffer = 5.0;
+	private int warningDistance = 5;
+	private int warningTime = 15;
+	private double centerX = 0;
+	private double centerZ = 0;
+
+	public WorldBorderMock(World world, Server server)
+	{
+		this.world = world;
+		this.server = server;
+	}
+
+	@Override
+	public void reset()
+	{
+		setSize(6.0E7);
```

Comment:
`6.0E7` is used twice here, I think there should be a constant for this at the top of the class.
`private static final double DEFAULT_BORDER_SIZE = 6.0E7`

Avoiding magic numbers like these makes it easier to tweak them if this default size changes in future versions of Minecraft.

*Same theoretically goes for the other values.*